src/timesync/timesyncd.c

```diff
@@ -22,9 +22,8 @@
 #include "timesyncd-manager.h"
 #include "user-util.h"
 
-static int advance_tstamp(int fd, const struct stat *st) {
+static int advance_tstamp(int fd, usec_t min) {
```

Comment:
i'd probably not call this "min", since various libraries have macros of such a name